solutions/saas-microservices/README.md

```diff
@@ -0,0 +1,28 @@
+# SaaS Microservices
+
+This is an example SaaS project with 2 API services defined as microservices.
+
+## Getting Started
+
+```sh
+pnpm install
+pnpm dev
+```
+
+Open http://localhost:3024 to view the dashboard. The dashboard will make API calls
+to the `users` and `dashboard` microservices for the information to display in the
+dashboard.
+
+## How It Works
+
+There are 3 separate applications in this example:
+
+- `dashboard` - A [Next.js](https://nextjs.org/) application to show the UI.
```

Comment:
should we mention this is the "router" or entrypoint? i'm guessing it is since it has the microfrontends.json file but I believe it could help understanding the high level overview